src/Tizen.NUI/src/internal/Layouting/GridLocations.cs

```diff
@@ -125,5 +125,49 @@ public List<Cell> GetLocations()
                 y2 = y2 + rowHeight;
             }
         }
+
+
+        /// <summary>
+        /// [Draft] Uses the given parameters to calculate the x,y coordinates of each cell and cell size.
+        /// </summary>
+         public void CalculateLocationsRow( int numberOfRows, int availableWidth,
```

Comment:
Wrong code styles. Use Code Formatter with default configuration of VS. 
Edit -> Advanced -> Format Document or Format Selection